sale_order_report_without_price/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 {
     "name": "Sale Order Report Without Price",
     "summary": "Allow you to generate quotation and order reports without price.",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
```

Comment:
@santostelmo You are missing migration commit :sweat_smile:

Reply:
That change was automatically applied when I executed pre-commit

Reply:
I renamed the commit

Reply:
Hello @santostelmo locally tested :+1: 

About commit history question, @rousseldenis if 100 % of changes are finally made at first pre-commit step, is correct if commit history only adds `[IMP] $module: pre-commit auto fixes` commit? If it's right, the only thing still needed is renaming existing commit again, isn't it?

Reply:
In fact, you didn't follow the migration guide exactly: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#how-to

The important thing is to use `--no-verify` when applying pre-commit. It will let the former module version but will apply the formatting.

So, then, you need to change module version in a further commit.

Could you retry ?